### DISCUSSION TITLE
Getting replex working on Unraid

### DISCUSSION THREAD

**Discussion by sdavisNL** [lostb1t/replex#164] [General]
I know the Unraid docker container is unsupported, but I'm just wondering if anyone has gotten this working with both Plex and replex as Unraid Docker containers?

The readme states Plex shouldn't be running in Host mode, but I'm not sure what the alternative is. Using bridge mode breaks Plex. I'm a docker/unraid newbie so any help would be appreciated!

**Comment by lostb1t** [comment]
Why would it be unsupported? Theres an unraid template in the following repo: https://github.com/selfhosters/unRAID-CA-templates did you give that a try?

**Reply by sdavisNL** [reply]
Yeah, that's the one I'm using. I thought I had read it wasn't supported, but may have mixed that up with another project. 

I have it installed, I can access PMS via the replex port, but everything on the homescreen across all clients remains the same. I do run Plex as an Unraid docker container so it's running in Host mode, which the readme says is a no-go. Just wondering if anyone has any tips to get it working.

**Reply by lostb1t** [reply]
yes because its in host mode theres a big change it uses the local network adress. Try using mobile data on your phone, if setup correctly it should then work on your phone. (that way you know what the issue is)

**Reply by lostb1t** [reply]
And what do you mean with "breaks"

bridge mode should work if setup correctly.

**Reply by sdavisNL** [reply]
Doesn't seem like I have it setup correctly as accessing it remotely gives me the same home screen. These are my debug logs:

>  text  error  warn  system  array  login  
> 
> 2024-09-19T17:53:46.094671Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=706.225152ms
> 2024-09-19T17:53:46.163511Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=775.041862ms
> 2024-09-19T17:53:46.209149Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=820.622389ms
> 2024-09-19T17:53:46.209696Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=372.402422ms
> 2024-09-19T17:53:46.443344Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=258.552281ms
> 2024-09-19T17:53:46.518588Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=700.745738ms
> 2024-09-19T17:53:46.520780Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=401.058986ms
> 2024-09-19T17:53:46.730933Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=893.61839ms
> 2024-09-19T17:53:50.315420Z DEBUG replex::logging: Response status=200 OK path=/web/js/chunk-8394-ee2e62e11943397e59bd-plex-4.136.1-8affb58.js duration=24.390786ms
> 2024-09-19T17:53:50.315715Z DEBUG replex::logging: Response status=200 OK path=/web/js/chunk-2825-09c4a3b6b400d8a5fe45-plex-4.136.1-8affb58.js duration=24.893148ms
> 2024-09-19T17:53:50.317709Z DEBUG replex::logging: Response status=200 OK path=/web/chunk-8394-ee2e62e11943397e59bd-plex-4.136.1-8affb58.css duration=26.897368ms
> 2024-09-19T17:53:50.333229Z DEBUG replex::logging: Response status=200 OK path=/web/js/chunk-4861-182789aac86a49bd0392-plex-4.136.1-8affb58.js duration=23.567213ms
> 2024-09-19T17:54:30.322997Z DEBUG replex::logging: Response status=200 OK path=/web/js/chunk-9769-698f56337b72c8e1b525-plex-4.136.1-8affb58.js duration=28.903525ms
> 2024-09-19T17:54:30.341633Z DEBUG replex::logging: Response status=200 OK path=/web/chunk-3126-1f7fc252c7993bdd35ef-plex-4.136.1-8affb58.css duration=26.920808ms
> 2024-09-19T17:54:30.341985Z DEBUG replex::logging: Response status=200 OK path=/web/chunk-2622-63b2aaa4850d6cb882f2-plex-4.136.1-8affb58.css duration=27.344548ms
> 2024-09-19T17:54:30.344331Z DEBUG replex::logging: Response status=200 OK path=/web/js/chunk-2622-63b2aaa4850d6cb882f2-plex-4.136.1-8affb58.js duration=29.599765ms
> 2024-09-19T17:54:30.346208Z DEBUG replex::logging: Response status=200 OK path=/web/js/chunk-3126-1f7fc252c7993bdd35ef-plex-4.136.1-8affb58.js duration=31.568525ms
> 2024-09-19T18:08:00.427279Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=245.25251ms
> 2024-09-19T18:08:00.458783Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=248.422539ms
> 2024-09-19T18:08:00.531766Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=38.213729ms
> 2024-09-19T18:08:00.551562Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=23.727482ms
> 2024-09-19T18:08:00.554737Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=26.920501ms
> 2024-09-19T18:08:01.009319Z DEBUG replex::logging: Response status=200 OK path=/playlists/all duration=390.858759ms
> 2024-09-19T18:08:01.467105Z DEBUG replex::logging: Response status=200 OK path=/library/sections/41/all duration=809.366599ms
> 2024-09-19T18:08:01.985416Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=478.841997ms
> 2024-09-19T18:08:02.707471Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/247657/children duration=2.048890982s
> 2024-09-19T18:08:03.108596Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/136430/children duration=2.44990199s
> 2024-09-19T18:08:03.495745Z DEBUG replex::logging: Response status=200 OK path=/library/sections/6/all duration=1.493389132s
> 2024-09-19T18:08:04.593708Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=1.86207458s
> 2024-09-19T18:08:05.473482Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/241663/children duration=4.816023348s
> 2024-09-19T18:08:05.635691Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=4.606111491s
> 2024-09-19T18:08:05.675331Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/241664/children duration=5.017762571s
> 2024-09-19T18:20:31.503202Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=112.042645ms
> 2024-09-19T18:20:31.726647Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/247657/children duration=26.554216ms
> 2024-09-19T18:20:31.756302Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/136430/children duration=56.301164ms
> 2024-09-19T18:20:31.804228Z DEBUG replex::logging: Response status=200 OK path=/playlists/all duration=139.79578ms
> 2024-09-19T18:20:31.824854Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/241663/children duration=124.846017ms
> 2024-09-19T18:20:31.842777Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/241664/children duration=142.594827ms
> 2024-09-19T18:20:31.937170Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=21.94068ms
> 2024-09-19T18:20:32.244285Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=467.350304ms
> 2024-09-19T18:20:32.417830Z DEBUG replex::logging: Response status=200 OK path=/library/sections/41/all duration=718.223919ms
> 2024-09-19T18:20:32.810947Z DEBUG replex::logging: Response status=200 OK path=/library/sections/6/all duration=993.990243ms
> 2024-09-19T18:20:33.135040Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=1.244964138s
> 2024-09-19T18:20:34.853932Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=3.117467833s
> 2024-09-19T18:34:34.185072Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=102.616643ms
> 2024-09-19T18:34:39.060434Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/241663/children duration=123.447323ms
> 2024-09-19T18:34:39.092064Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/247657/children duration=31.144411ms
> 2024-09-19T18:34:39.128085Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/136430/children duration=38.430429ms
> 2024-09-19T18:34:39.194815Z DEBUG replex::logging: Response status=200 OK path=/playlists/all duration=133.868633ms
> 2024-09-19T18:34:39.204507Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/241664/children duration=142.760793ms
> 2024-09-19T18:34:39.579612Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=518.602092ms
> 2024-09-19T18:34:39.726827Z DEBUG replex::logging: Response status=200 OK path=/library/sections/41/all duration=665.683759ms
> 2024-09-19T18:34:40.160242Z DEBUG replex::logging: Response status=200 OK path=/library/sections/6/all duration=1.057042129s
> 2024-09-19T18:34:40.560250Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=1.358780954s
> 2024-09-19T18:34:42.278346Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=3.142603695s
> 2024-09-19T18:49:24.426620Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=118.016367ms
> 2024-09-19T18:49:27.623680Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=409.424805ms
> 2024-09-19T18:49:27.651114Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=390.637094ms
> 2024-09-19T18:49:27.690771Z DEBUG replex::logging: Response status=200 OK path=/library/sections/2/firstCharacter duration=3.174792611s
> 2024-09-19T18:49:29.394201Z DEBUG replex::logging: Response status=200 OK path=/library/sections/2/all duration=4.897746596s
> 2024-09-19T18:53:02.627546Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=105.441837ms
> 2024-09-19T19:03:03.958056Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=140.291554ms
> 2024-09-19T19:03:20.137190Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=184.828239ms
> 2024-09-19T19:05:21.679917Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=122.771363ms
> 2024-09-19T19:05:22.077418Z DEBUG replex::logging: Response status=200 OK path=/playlists/all duration=267.063268ms
> 2024-09-19T19:05:22.247034Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/247657/children duration=417.23599ms
> 2024-09-19T19:05:22.471534Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/136430/children duration=641.783669ms
> 2024-09-19T19:05:22.728580Z DEBUG replex::logging: Response status=200 OK path=/library/sections/41/all duration=898.902464ms
> 2024-09-19T19:05:23.268282Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=1.009365395s
> 2024-09-19T19:05:23.301107Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=22.816127ms
> 2024-09-19T19:05:23.743468Z DEBUG replex::logging: Response status=200 OK path=/library/sections/6/all duration=1.261890306s
> 2024-09-19T19:05:24.577067Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=1.822974744s
> 2024-09-19T19:05:24.961300Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/241663/children duration=3.131753714s
> 2024-09-19T19:05:25.067670Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/241664/children duration=3.237823099s
> 2024-09-19T19:05:26.243719Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=4.159715339s
> 2024-09-19T19:12:53.773955Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=141.24188ms
> 2024-09-19T19:14:00.921788Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=101.283945ms
> 2024-09-19T19:14:01.115938Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/136430/children duration=32.448043ms
> 2024-09-19T19:14:01.131527Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/247657/children duration=48.169828ms
> 2024-09-19T19:14:01.178266Z DEBUG replex::logging: Response status=200 OK path=/playlists/all duration=116.696054ms
> 2024-09-19T19:14:01.204289Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/241664/children duration=121.002091ms
> 2024-09-19T19:14:01.233254Z DEBUG replex::logging: Response status=200 OK path=/replex/shelf/library/collections/241663/children duration=149.979459ms
> 2024-09-19T19:14:01.289477Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=21.139344ms
> 2024-09-19T19:14:01.324995Z DEBUG replex::logging: Response status=200 OK path=/photo/:/transcode duration=20.263682ms
> 2024-09-19T19:14:01.685407Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=506.277295ms
> 2024-09-19T19:14:01.758010Z DEBUG replex::logging: Response status=200 OK path=/library/sections/41/all duration=674.768311ms
> 2024-09-19T19:14:02.178957Z DEBUG replex::logging: Response status=200 OK path=/library/sections/6/all duration=979.897392ms
> 2024-09-19T19:14:02.485293Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=1.248022549s
> 2024-09-19T19:14:04.105470Z DEBUG replex::logging: Response status=200 OK path=/hubs/home/recentlyAdded duration=2.926341855s
> 2024-09-19T19:36:07.917844Z DEBUG replex::logging: Response status=200 OK path=/media/providers duration=174.17259ms
>

**Reply by lostb1t** [reply]
and you followed these instructions: https://github.com/lostb1t/replex?tab=readme-ov-file#remote-access-force-clients-to-use-the-proxy

**Reply by sdavisNL** [reply]
Question: If I have it setup correctly without the remote access part done, shouldn't replex rows show up at local:3001?

**Reply by lostb1t** [reply]
yes and no, yes plex should show up on the replex port but if in host mode plex will probaply try to resolve the api calls to its own local adress

you can see what adress plex is calling by opening the web inspector and under network look at the xhr requests.

**Reply by sdavisNL** [reply]
Do you know of a guide to set Plex up in something other than host mode. I switch to bridge mode and Plex stops working, like I said new to all this.

**Reply by lostb1t** [reply]
> and you followed these instructions: https://github.com/lostb1t/replex?tab=readme-ov-file#remote-access-force-clients-to-use-the-proxy

You didn't answer this. Did you follow the instructions and setup a domain?
